source/_introduction.html.md.erb

```diff
@@ -298,9 +298,38 @@ The JSON representation of the objects returned from the List All REST endpoints
 Smartsheet will add new functionality and bug fixes to the API over time. Make sure that your code can handle new JSON properties gracefully. 
 Also, make sure your code does not depend on the order in which JSON objects are returned, unless it is explicitly stated in this documentation. 
 
+In some cases, such as when new column types or new widgets are added, Smartsheet offers a backwards-compatibility option as well as giving you access to the new functionality. For instance, if a sheet has column types of MULTI_CONTACT_LIST or MULTI_PICKLIST, there are two ways of receiving the response:
+
+* As text (column type TEXT_NUMBER)
+* As an array of values (column types of MULTI_CONTACT_LIST or MULTI_PICKLIST)
+
+Smartsheet lets you choose how and when to upgrade your queries and response values by using a "level" query parameter as a differentiator, for instance "?level=2". There are also incremented values in the Column object's "version" parameter. However, level and version vary by whether you are calling a report, sheet, or Sight (aka dashboard) and by what data you want to return.
```

Comment:
"There are also incremented values in the Column object's "version" parameter" 

This is confusing to me. First off, I am not sure what the column object's version parameter is. I am pretty sure you are referring to the "level" query parameter again?

Reply:
I think he probably means `version` property. @kfansler?

Reply:
Will double check this. I am guessing that the returned column object will return the restDataType level it used

Reply:
do you know what the version property does @timwellswa ?